src/igvfd/types/open_reading_frame.py

```diff
@@ -0,0 +1,25 @@
+from snovault import (
+    collection,
+    load_schema,
+)
+from .base import (
+    Item
+)
+from snovault.util import Path
+
+
+@collection(
+    name='open-reading-frame',
+    unique_key='open_reading_frame:orf_id',
+    properties={
+        'title': 'Open Reading Frame',
+        'description': 'Protein-encoding open reading frames (ORF)',
+    }
+)
+class OpenReadingFrame(Item):
+    item_type = 'open_reading_frame'
+    schema = load_schema('igvfd:schemas/open_reading_frame.json')
+    name_key = 'orf_id'
+    embedded_with_frame = [
+        Path('gene', include=['@id', 'symbol']),
+    ]
```

Comment:
I think for there to be an additional unique key there needs to be a special calculated property here. There are some examples in Institutional Certificate, Ontology Term, or Biomarker

Reply:
done

Reply:
undone this. Turns out we can just use name_key. No calculated property needed because uniqueness from orf_id is pretty simple